### PR TITLE
Hardcode plan c message

### PR DESCRIPTION
**Pull request by EVogel1999**
Hardcodes a plan c submission banner.  Merge after the banner fix is in.

Message says: Clark is now accepting curriculum submissions to the Plan C collection.

Pic:
Screen Shot 2020-03-17 at 9 43 20 AM

### CODE REVIEW

File: src/app/core/messages.service.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ export class Message {
 }
 @Injectable()
 export class MessagesService {
-  private _message: Message;
+  private _message: Message = new Message(true, 'Clark is now accepting curriculum submissions to the Plan C collection.');
```
Comment (pzalep1) [inline src/app/core/messages.service.ts]: We can make Plan C a link to the collection page.

Comment (EVogel1999) [reply src/app/core/messages.service.ts]: How to I add a link?  I tried doing embedded <a> tag but it didn't convert to a link.

**Comment by EVogel1999** [comment]
Pic with link:

**Comment by EVogel1999** [comment]
Picture with link:
Screen Shot 2020-03-17 at 10 17 49 AM

**Pull request merged**